src/data/seeds/users.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.seed = async function (knex) {
+  // Deletes ALL existing entries
+  await knex('usuario').del()
+  await knex('usuario').insert([
+    {
+      id: 1,
+      nome: 'admin001',
```

Comment:
```suggestion
      nome: 'Administrador',
```
Esse é o nome ou o `"username"` de login?

Reply:
Era o username, mas com a adição do campo nome, será o nome.